js/ftx.js

```diff
@@ -683,7 +683,7 @@ module.exports = class ftx extends Exchange {
             this.safeNumber (ohlcv, 'high'),
             this.safeNumber (ohlcv, 'low'),
             this.safeNumber (ohlcv, 'close'),
-            this.safeNumber (ohlcv, 'volume'),
+            this.safeNumber (ohlcv, 'volume', 0),
```

Comment:
i don't think we should default to 0 here

Reply:
everything else does

Reply:
right, but in this case I don't think we should do it

Reply:
there's two reasons, firstly in ccxt if something is not defined we usually don't default it, and secondly 0 is a float / integer and not a `number` type

Reply:
okex doesn't

Reply:
I would agree, we should not default to 0 here.